mne/channels/montage.py

```diff
@@ -718,6 +723,44 @@ def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
         fids = [fids[key] for key in ('nasion', 'lpa', 'rpa')]
         coord_frame = 'unknown'
 
+    elif bvct is not None:
+        if not all(x is None for x in (hsp, hpi, elp, point_names, fif, egi)):
+            raise ValueError('hsp, hpi, elp, point_names, fif, egi must all be '
+                             'None if bvct is not None')
+        _check_fname(bvct, overwrite='read', must_exist=True)
+
+        root = ElementTree.parse(bvct).getroot()
+        sensors = root.find('CapTrakElectrodeList')
+
+        fids = {}
+        dig_ch_pos = {}
+
+        fid_name_map = {'Nasion': 'nasion','RPA': 'rpa','LPA': 'lpa'}
+
+        scale = dict(mm=1e-3, cm=1e-2, auto=1e-3, m=1)
```

Comment:
You seem to assume here, that the CapTrack device measures electrode locations in units of meters. However, according to the "help" in the CapTrack computer, the distances are specified in millimeters (mm).

Unfortunately, this is not specified in the manual. But I happen to have a captrack device and I checked. I also made an entry for this in the [fieldtrip wiki](http://www.fieldtriptoolbox.org/faq/how_are_the_different_head_and_mri_coordinate_systems_defined/#details-of-the-captrack-coordinate-system)

Reply:
Ah okay, I see now, this is right. We always scale to meters ... and with `'auto'`, we assume mm :+1: